Dev fp vs oop/README.md

```diff
@@ -0,0 +1,107 @@
+# Contrast functional patterns vs. OOP design patterns
+Sumiit have worked on the README file, but I want to do more with the code. Ashley suggested some areas to focus on. I want to make sure I'm helping the project in the right way so summit have worked on javascript in his part of contribution and i am working on JAVA 
```

Comment:
You can remove this section. Including the motivation for your contribution is not necessary.